workspace-manager-clienttests/render-config.sh

```diff
@@ -7,16 +7,21 @@ USER_DELEGATED_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/firecloud/dev/common/firec
 USER_DELEGATED_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/rendered/user-delegated-service-account.json
 TESTRUNNER_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/common/testrunner-sa
 TESTRUNNER_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/rendered/testrunner-service-account.json
+TESTRUNNER_IAM_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/ichang/testrunner-sa
```

Comment:
is this the right Vault path, including the "ichang"?

Reply:
The resiliency tests will be restricted to either a namespace (personal environment like `ichang`) or preview namespace environment (when it becomes available). The test runner SA currently only exist under the `ichang` path. The [Test Runner Service Account](https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM/edit?usp=sharing) document provides `devOps` guidelines on terraforming test runner SAs and RBAC roles for namespaces.  Resiliency tests currently targeted `ichang` because that's the only namespace where k8s RBAC role has been setup for testing.